### ISSUE TITLE
Entry points not registered even when using reentry_register=True in setup.py

### ISSUE THREAD

**Issue by sphuber** [DropD/reentry#16]
This was discovered due to tests failing on the `aiida-quantumespresso` repository. The `setup.py` does contain the `reentry_register=True` directive, but still after the plugin is installed and the `before_script.sh` attempts to setup some codes through `verdi code setup` the necessary entry points of the plugin itself are not found by the entrypoint manager. I could hack in a fix by installing `reentry` before installing the package, which made sure that the entrypoints were visible. However, then the problem was that the `aiida-core` distribution could not be found by the entrypoint manager. That is to say calling:

    from reentry import manager as epm
    epm.get_entry_map('aiida-core')

would lead to the following exception:

```
ValueError                                Traceback (most recent call last)
<ipython-input-8-b39593196a02> in <module>()
----> 1 l = [g for g in epm.get_entry_map('aiida-core').iterkeys()]

/home/sphuber/.virtualenvs/aiida_qe/local/lib/python2.7/site-packages/reentry/manager.pyc in get_entry_map(distname, group)
     27     The backend may only load pkg_resources if any of the entry points contain extras requirements
     28     """
---> 29     return bkend.get_map(dist=distname, group=group)
     30 
     31 

/home/sphuber/.virtualenvs/aiida_qe/local/lib/python2.7/site-packages/reentry/jsonbackend.pyc in get_map(self, dist, group, name)
    172         '''sanitize dist kwarg'''
    173         if dist not in self.epmap:
--> 174             raise ValueError("The {} distribution was not found.".format(dist))
    175 
    176         '''sanitize groups kwarg'''

ValueError: The aiida-core distribution was not found.
```

Again I could temporarily circumvent this problem by calling `reentry scan` explicitly after installing the plugin, but it does mean that for users installing it through `pip install aiida-quantumespresso` they will also have to do this, which we would rather prevent of course

**Comment by ltalirz** [comment]
I can certainly confirm that I had to `reentry scan -r aiida` after installing plugins.
That's why, for the moment, it's mentioned in the docs
http://aiida-core.readthedocs.io/en/release_v0.10.0/installation/plugins.html

Once this issue is resolved, we can hopefully remove these lines from there.

**Comment by DropD** [comment]
fixed in v1.1.1
